src/data/bitvec/basic.lean

```diff
@@ -34,20 +34,19 @@ by rw [list.foldr_reverse, flip]; refl
 
 lemma to_nat_lt {n : ℕ} (v : bitvec n) : v.to_nat < 2^n :=
 begin
-  suffices : v.to_nat + 1 ≤ 2 ^ n, { simpa },
+  change v.to_nat + 1 ≤ 2 ^ n,
```

Comment:
this doesn't belong to this PR.

Reply:
Do you mean this line or this file? If the latter, then the answer is "it does": `mono` made more progress because `mul_le_mul'` is now a `mono` lemma. While fixing a failure, I've got a timeout (not sure why). Fixed it by avoiding all potentially slow tactics.

Reply:
I meant the file. Thanks for the clarification.